### ISSUE TITLE
Неотвечает

### ISSUE THREAD

**Issue by devcline** [VKCOM/bot-example-php#2]
Почему то все прошло гладко а бот то неотвечает
Подразумеваю что нету secretkey как у вас в другом примере handler?

**Comment by tsivarev** [comment]
А токен получен с правами messages и photos?

**Comment by devcline** [comment]
Немножко вас не понял: Токен ведь от сообщества? В секции Работа С API
Создать ключ: Разрешить приложению доступ к сообщениям сообщества
И он выдает толкен? Или нужен тот что выдается через ссылку? С профиля https://oauth.vk.com/authorize?client_id=..............................

Спасибо

**Comment by tsivarev** [comment]
Да, от сообщества. Все верно. Только ещё поставьте галочку напротив доступа к фотографиям сообщества.

**Comment by devcline** [comment]
Все супер:) Спасибо:) Я так понял одновременно если создать два сервера с двумя ботами из Callback API , то будет работать только один? Отвечает только первый. Но по логам запросов из сообщества видно что скрипт реагирует.

**Comment by APTEMOH** [comment]
> Да, от сообщества. Все верно. Только ещё поставьте галочку напротив доступа к фотографиям сообщества.

Это где? Токен же в группе (в настройках выдается)
